google-cloud-logging/lib/google-cloud-logging.rb

```diff
@@ -139,17 +137,24 @@ def self.logging project_id = nil, credentials = nil, scope: nil,
       "LOGGING_CREDENTIALS", "LOGGING_CREDENTIALS_JSON",
       "LOGGING_KEYFILE", "LOGGING_KEYFILE_JSON"
   end
+  default_scopes = [
+    "https://www.googleapis.com/auth/cloud-platform",
+    "https://www.googleapis.com/auth/cloud-platform.read-only",
+    "https://www.googleapis.com/auth/logging.admin",
+    "https://www.googleapis.com/auth/logging.read",
+    "https://www.googleapis.com/auth/logging.write"
+  ]
 
   config.add_field! :project_id, default_project, match: String, allow_nil: true
   config.add_alias! :project, :project_id
   config.add_field! :credentials, default_creds,
                     match:     [String, Hash, Google::Auth::Credentials],
                     allow_nil: true
   config.add_alias! :keyfile, :credentials
-  config.add_field! :scope, nil, match: [String, Array]
+  config.add_field! :scope, default_scopes, match: [String, Array]
   config.add_field! :timeout, nil, match: Integer
   config.add_field! :client_config, nil, match: Hash
```

Comment:
Can you remove this `client_config` usage as well?

Reply:
No, that _would_ be a breaking change, removing the field from the public config object.

Reply:
`client_config` is also a keyword arg in the factory methods in this class.

Reply:
Again, those are public, and removing them would be a breaking change. Maybe we're okay with that, given that the plan is to release these as semver-major, though.

Reply:
Removed.